lms/static/coffee/src/customwmd.coffee

```diff
@@ -2,10 +2,8 @@
 
 $ ->
 
-  if not MathJax?
-    return
-
-  HUB = MathJax.Hub
+  if MathJax?
+    HUB = MathJax.Hub
```

Comment:
I added it later
https://github.com/edx/edx-platform/pull/6775/files#diff-0337bafafc5a0c3e2d96415fc15e236fR118